sleap_nn/train.py

```
@@ -440,9 +440,10 @@ def get_trainer_config(
     optimizer: str = "Adam",
     learning_rate: float = 1e-3,
     amsgrad: bool = False,
-    lr_scheduler: Optional[
-        Union[str, Dict[str, Any]]
-    ] = None,  # {"lr_scheduler": {"step_lr": {}, "reduce_lr_on_plateau": {}}}
+    lr_scheduler: Union[str, Dict[str, Any]] = {
+        "step_lr": None,
+        "reduce_lr_on_plateau": None,
+    },
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)